ext/skeleton/skeleton.stub.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
-function %EXTNAME%_test1(): void {}
```

Comment:
I am not sure if it's ok to get rid of the %EXTNAME% prefix, but it definitely makes it impossible to generate all the stubs

Reply:
Don't see an issue with this.